### DISCUSSION TITLE
Forked playframework error when launching sbt

### DISCUSSION THREAD

**Discussion by jausmann** [playframework/playframework#11395] [Q&A] [answered]
This error is given when launching sbt with a forked play framework.

[error] Failed to derive version from git tags. Maybe run `git fetch --unshallow`? Version: 0.0.0+12149-edf094af+20220819-1128-SNAPSHOT

**Comment by francisdb** [comment]
Check `project/plugins.sbt`
This sounds like an error from the `git` or the `sbt-dynver` plugin, maybe commenting those out helps. 

Did you download the repo as zip instead of cloning using git?

**Reply by jausmann** [reply]
The repo was cloned through git.

**Reply by jausmann** [reply]
```
// Copyright (C) Lightbend Inc. <https://www.lightbend.com>

enablePlugins(BuildInfoPlugin)

// when updating sbtNativePackager version, be sure to also update the documentation links in
// documentation/manual/working/commonGuide/production/Deploying.md
val sbtNativePackager  = "1.9.9"
val mima               = "1.1.0"
val sbtJavaFormatter   = "0.7.0"
val sbtJmh             = "0.4.3"
val webjarsLocatorCore = "0.50"
val sbtHeader          = "5.7.0"
val scalafmt           = "2.0.1"
val sbtTwirl: String   = sys.props.getOrElse("twirl.version", "1.6.0-M6") // sync with documentation/project/plugins.sbt
val interplay: String  = sys.props.getOrElse("interplay.version", "3.0.5")

buildInfoKeys := Seq[BuildInfoKey](
  "sbtNativePackagerVersion" -> sbtNativePackager,
  "sbtTwirlVersion"          -> sbtTwirl,
)

logLevel := Level.Warn

scalacOptions ++= Seq("-deprecation", "-language:_")

addSbtPlugin("com.typesafe.play"  % "interplay"             % interplay)
addSbtPlugin("com.typesafe.play"  % "sbt-twirl"             % sbtTwirl)
addSbtPlugin("com.typesafe"       % "sbt-mima-plugin"       % mima)
addSbtPlugin("com.lightbend.sbt"  % "sbt-bill-of-materials" % "1.0.2")
addSbtPlugin("com.lightbend.sbt"  % "sbt-java-formatter"    % sbtJavaFormatter)
addSbtPlugin("pl.project13.scala" % "sbt-jmh"               % sbtJmh)
addSbtPlugin("de.heikoseeberger"  % "sbt-header"            % sbtHeader)
addSbtPlugin("org.scalameta"      % "sbt-scalafmt"          % scalafmt)
addSbtPlugin("com.github.sbt"     % "sbt-ci-release"        % "1.5.10")

addSbtPlugin("com.lightbend.akka" % "sbt-akka-version-check" % "0.1")

libraryDependencies ++= Seq(
  "org.webjars" % "webjars-locator-core" % webjarsLocatorCore
)

resolvers += Resolver.typesafeRepo("releases")
```

**Comment by francisdb** [comment]
Yeah, sbt-ci-release pulls in dynver I think.
Did you actually try the `git fetch --unshallow`? A fork should contain the tags.

“git fetch” with the “–all” and the “–tags” options should make sure you have them locally.

**Reply by francisdb** [reply]
`git fetch upstream --tags` might also help if the tags are missing on your fork

**Reply by jausmann** [reply]
Thanks for the ideas to try. However none of them have resolved the issue.

**Comment by francisdb** [comment, marked as answer]
A fork on github has this by default enabled:

<img width="597" alt="image" src="https://user-images.githubusercontent.com/161305/186149225-79d28559-d1fd-4b44-9eb1-60289650528f.png">

Also there are zero tags copied in this case

<img width="532" alt="image" src="https://user-images.githubusercontent.com/161305/186149337-f4d19542-2090-4791-860c-05fa64dc405f.png">

disabling that option when forking gives you this

<img width="586" alt="image" src="https://user-images.githubusercontent.com/161305/186149624-17c157f0-93ec-4081-8bee-c28986cd233b.png">

**Reply by jausmann** [reply]
That worked great. I deleted my forked repo then forked it again with that option disabled.
